### DISCUSSION TITLE
How to use this toast inside the class component

### DISCUSSION THREAD

**Discussion by Mathivanan1803** [arnnis/react-native-fast-toast#151] [Q&A] [answered]
Hey, Greetings of the day. I tried to use this toast within the class component by implementing higher-order component. But while using this am getting error like undefined is not an object("this.props.toas
t.show"). Help me to fix this issue. thanks in advance.

With this question i  attached my HOC
<img width="360" alt="Screenshot 2022-08-08 185453" src="https://user-images.githubusercontent.com/45687516/183428537-18086f83-efa3-4666-b2c7-7278ac5fe3d7.png">

**Comment by Kajanan02** [comment, marked as answer]
you can try this way this.props.show("msg"), It's work for me.
